src/templates/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mywebsite</title>
+  </head>
+  <link rel="stylesheet" href="../css/style.css" />
```

Comment:
make sure to add bootstrap here and this should allow you to work with it

Reply:
Also, look into templating the header footer and other parts of the pages that will be reused.